WebDriverAgentLib/Utilities/XCTestPrivateSymbols.m

```diff
@@ -56,8 +55,9 @@
 
 NSArray<NSString*> *FBStandardAttributeNames(void)
 {
-  return [XCElementSnapshot sanitizedElementSnapshotHierarchyAttributesForAttributes:nil
-                                                                             isMacOS:NO];
+  Class xcElementSnapshotClass = NSClassFromString(@"XCElementSnapshot");
```

Comment:
Perhaps we should use dispatch_once here as well?

Reply:
will do